src/nyc_trees/nyc_trees/settings/base.py

```diff
@@ -286,3 +286,5 @@
 # If geocoding a string produces no results, this string will be
 # appended for a second attempt.
 GEOCODE_FALLBACK_SUFFIX = ', New York, NY'
+
+TILER_URL = environ['TILER_HOST']
```

Comment:
All of the other environmental variables pulled in this settings file use `environ.get` with some default. `localhost` seems like the best option for a default value.